sale_partner_approval/data/exception_rule.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="excep_partner_approved" model="exception.rule">
+        <field name="name">Customer is Approved</field>
+        <field name="description">Customer must be a customer approved for Sales</field>
+        <field name="model">sale.order</field>
+        <field name="exception_type">by_domain</field>
+        <field name="domain">[('partner_id.sale_ok', '=', False)]</field>
+        <field name="is_blocking">True</field>
```

Comment:
I am not able to install the module due to this line which is causing the following error: 
ValueError: Invalid field 'is_blocking' on model 'exception.rule'
Is this missing a dependancy?

Reply:
my base_exception module was out of date, updated that and will test.